src/Uno.UI.Wasm/ts/MonoSupport.ts

```diff
@@ -7,7 +9,7 @@
 	export class jsCallDispatcher {
 
 		static registrations: Map<string, any> = new Map<string, any>();
-		static methodMap: { [id: number]: any } = {};
+		static methodMap: { [id: string]: any } = {};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class property 'methodMap' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5247895)